### DISCUSSION TITLE
Make a headless client for testing

### DISCUSSION THREAD

**Discussion by godwhoa** [piqueserver/piqueserver#660] [Feature Requests]

**Comment by xtreme8000** [comment]
I could adapt BetterSpades to allow for this

**Comment by NotAFile** [comment]
We've discussed modifying BS before, but I wonder if that's really useful. If we want to make it headless, we basically remove all of the sound and graphics code. Then the only thing left are the packet definitions and physics, both of which are in pyspades already. So it would probably be more work to make BetterSpades code accessible from python than to just use the existing pyspades code.
